e2e-tests/tests/gr-one-pod/00-deploy-operator.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |-
+      set -o errexit
+      set -o xtrace
+
+      source ../../functions
+
+      apply_s3_storage_secrets
+      deploy_operator
+      deploy_non_tls_cluster_secrets
+      deploy_tls_cluster_secrets
+      deploy_client
+      deploy_minio
+    timeout: 300
```

Comment:
```suggestion
      timeout: 300
```

Reply:
I think it needs to be indented this way to be applied to `script` block.